ocpp-v1_6/src/main/java/eu/chargetime/ocpp/WebSocketListener.java

```diff
@@ -132,6 +132,9 @@ public void onStart() {
         }
 
         server.setConnectionLostTimeout(pingInterval);
+
+        server.setReuseAddr(true);
```

Comment:
That's what might solve all 'not connected' errors.

Reply:
@eupakhomov well, sometimes if you stops your bind-address operation, Linux core doesn't release a port, it is not according WS, that's core network stuff. I have no idea why 'false' state is a default state, so I vote to set is as 'true' by default, I do this all my life without any problems. @TVolden what do you think?